### DISCUSSION TITLE
Github APIs - 在 ClipCC 中探索 Github ！

### DISCUSSION THREAD

**Discussion by waterblock79** [Clipteam/clipcc-gui#47] [扩展发布/Extensions sharing]
**拓展英文名称**：Github APIs  
**拓展中文名称**：Github APIs
**关于拓展**：Github APIs 是一个可以在 ClipCC 中使用 Github 的官方 API 来访问 Github 的拓展。  
**直接链接**：[https://github.com/waterblock79/clipcc-github-apis/releases/download/0.1.0/com.extension.githubapis@0.1.0.ccx](https://github.com/waterblock79/clipcc-github-apis/releases/download/0.1.0/com.extension.githubapis@0.1.0.ccx)  

![image](https://user-images.githubusercontent.com/33573572/120910970-b8fe8400-c6b5-11eb-926e-20148f8acf77.png)

**Comment by waterblock79** [comment]
![image](https://user-images.githubusercontent.com/33573572/120911034-67a2c480-c6b6-11eb-835d-093cd7844b24.png)  
🤔@SinanGentoo

**Reply by SimonShiki** [reply]
我错了 这就改

**Reply by SimonShiki** [reply]
已帮助修改Labels

**Reply by waterblock79** [reply]
感谢
